MatrixSDK/Data/MXRoomState.m

```diff
@@ -591,6 +651,8 @@ - (id)copyWithZone:(NSZone *)zone
     // membership change (ex: "invited" -> "joined")
     stateCopy->members = [[NSMutableDictionary allocWithZone:zone] initWithDictionary:members];
 
+    stateCopy->thirdPartyInvites = [[NSMutableDictionary allocWithZone:zone] initWithDictionary:thirdPartyInvites];
+
     if (visibility)
```

Comment:
Is there a reason why 'membersWithThirdPartyInviteTokenCache' is not copied here?